Public broadcasting of Latvia.js

```diff
@@ -0,0 +1,335 @@
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2019 Mārtiņš Bruņenieks
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/raksts/')) {
+		return "newspaperArticle";
+	}
+	return false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "newspaperArticle") {
+		scrape(doc, url);
+	}
+}
+
+function formatDate(date) {
+	return date.toISOString().split('T')[0];
+}
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+
+	translator.setHandler('itemDone', function (obj, item) {
+		item.publicationTitle = ZU.xpathText(doc, '//section[@class="globalnav__item globalnav__item-sitename"]');
+		item.section = ZU.xpathText(doc, '//div[@class="article__info"]/section[3]');
+
+		item.creators = [];
+		
+		// Possible formats:
+		// Autori: Zanda Ozola-Balode (Panorāma), Linda Zalāne (Latvijas Radio Ziņu dienesta korespondente)
+		// Autori: Uģis Lībietis (Latvijas Radio ārzemju ziņu korespondents )
+		// Autori: Ludmila Glazunova
+		// Autori: LSM.lv ziņu redakcija
+		
+		var names = ZU.xpathText(doc, '//div[@class="article__info"]/section[@class="article__info__authors"]');
+		if (names) {
+			names = names.substring(names.indexOf(":") + 1);
+		
+			var authors = names.split(',');
+			
+			for (let author of authors) {
+				author = author.trim();
+				if (author === 'LSM.lv ziņu redakcija' || author === 'LETA' || author === 'BNS') {
+					item.creators.push({ lastName: author, creatorType: "author", fieldMode: true });
+				} else {
+					// Zanda Ozola-Balode (Panorāma)
+					if (author.indexOf('(') > 0) {
+						author = author.substring(0, author.indexOf('('));
+					}
+					item.creators.push(ZU.cleanAuthor(author, "author"));
+				}
+			}
+		}
+		
+		// Date string is of the format:
+		// Publicerat onsdag 15 november 2017 kl 00.11
+		// /html/body/main/section[8]/div/div[1]/section[1]/article/div[3]/section[1]
+		var dateString = ZU.xpathText(doc, '//div[@class="article__info"]/section[1]');
+		if (dateString.indexOf('Šodien') > 0) {
+			item.date = formatDate(new Date());
+		} else if (dateString.indexOf('Vakar') > 0) {
+			item.date = formatDate(new Date(new Date().setDate(new Date().getDate() - 1)));
```

Comment:
I guess this strings means today and yesterday. Please add an English comment for this. Moreover, the second one looks too complicated. I guess that similar things like in https://github.com/zotero/zotero/blob/eca2822651ac191752e0f055eb5c06d386dd5197/chrome/content/zotero/xpcom/date.js#L276 should work here.